pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -498,13 +509,15 @@ type DNSSDConfig struct {
 	//
 	// When set to NS, It requires Prometheus >= 2.49.0.
 	//
-	// +kubebuilder:validation:Enum=SRV;A;AAAA;MX;NS
+	// +kubebuilder:validation:Enum=A;AAAA;SRV;MX;NS
```

Comment:
nit: If you meant to sort alphabetically

```suggestion
	// +kubebuilder:validation:Enum=A;AAAA;MX;NS;SRV
```